### PR TITLE
Upgrade cypress test. Add check filter history.

### PR DESCRIPTION
**Pull request by DmitriyOparin**
Upgrade cypress test. Add check filter history.
Added new cssSelector for elements.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by ActiveChooN** [comment]
@DmitriyOparin, have thoughts about additional case for this tests. There is ability to select few filters that will be selected with `OR` operand. Can we add such case?
изображение

**Comment by DmitriyOparin** [comment]
OK, will try to do it.

**Comment by DmitriyOparin** [comment]
@ActiveChooN added new step for verify filters selected with OR operand. Now test failed due to incorrect display or is it correct display?

**Comment by ActiveChooN** [comment]
Guess last test case works just right. It select one track with `clientId==4` and two shapes `(label=="shape 3 points" & attr["polylines"]==true) | (label=="track 4 points" & width > 60)` and one of them are the same. I'm not sure, why are you comparing coordinates of filters tags. I meant that result of filtering with filters  `clientId==4` and `(label=="shape 3 points" & attr["polylines"]==true) | (label=="track 4 points" & width > 60)` should be the same as with the one filter `(clientId==4) | (label=="shape 3 points" & attr["polylines"]==true) | (label=="track 4 points" & width > 60)`.

**Comment by DmitriyOparin** [comment]
Thanks for explanation. Unnecessary verify removed.

File: tests/cypress/integration/actions_tasks_objects/case_18_filters_functionality.js [hunk 1]
```diff
@@ -103,46 +104,83 @@ context('Filters functionality.', () => {
...
-            );
+            const textFilter =
+                '(( label==["points shape"]) | (attr["type"]=="shape" & width > 50)) & (height > 50 & (clientID == serverID)))';
+            cvatFiltesList.push(textFilter);
+            cy.writeFilterValue(true, textFilter);
             checkingFilterApplication([]);
         });
+        it('Verify to show all filters', () => {
+            cvatFiltesList.forEach(function (filterValue) {
+                cy.contains('.cvat-annotations-filters-input-history-element', filterValue)
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_18_filters_functionality.js:164]: ``cy.contains`` should be enough. Isn't it?

File: tests/cypress/support/commands.js [hunk 2]
```diff
@@ -530,6 +529,28 @@ Cypress.Commands.add('writeFilterValue', (clear, filterValue) => {
...
+        cy.get('.cvat-annotations-filters-input').within(() => {
+            cy.get('.ant-select-selection-item-remove').click();
+        });
+    }
+    cy.get('body').click();
+    cy.get('.cvat-annotations-filters-input').click();
+    cy.contains('.cvat-annotations-filters-input-history-element', filterValue).scrollIntoView().click();
+    cy.get('body').click();
+    cy.get('.cvat-annotations-filters-input').within(() => {
+        cy.get('.ant-select-selection-item-content')
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:544]: Cannot we just check if ``.cvat-annotations-filters-input`` contains ``filterValue``? I am not sure why we need this additional array ``filterValues``.

Comment (DmitriyOparin) [reply tests/cypress/support/commands.js]: Yes, we can. Unnecessary actions removed.

File: tests/cypress/integration/actions_tasks_objects/case_18_filters_functionality.js (hunk 1 above)
Comment (DmitriyOparin) [reply tests/cypress/integration/actions_tasks_objects/case_18_filters_functionality.js]: Yes, sure. Unnecessary actions removed.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**